docs/core/whats-new/dotnet-9/libraries.md

```diff
@@ -527,11 +528,14 @@ The following code shows how to set the option (the `Book` type definition is sh
 
 :::code language="csharp" source="../snippets/dotnet-9/csharp/Serialization.cs" id="RespectNullable":::
 
-You can also enable this setting globally using the `System.Text.Json.JsonSerializerOptions.RespectNullableAnnotations` feature switch in your project file (for example, _.csproj_ file):
+> [!NOTE]
+> Due to how nullability annotations are represented in IL, the feature is restricted to annotations of non-generic properties. Please refer to the relevant documentation for more details.
```

Comment:
Any guesses on what that "relevant documentation" might be?

Reply:
It's a placeholder for the conceptual documentation we need to be adding at some point. I have created a draft containing all the relevant content.

Reply:
Should we comment that sentence out for now?

Reply:
Just the last sentence you mean? Sure.